src/components/core/blocks/Checkout/Shipping.vue

```diff
@@ -24,6 +24,10 @@ export default {
         this.$store.dispatch('checkout/updatePropValue', ['lastName', receivedData.lastName])
       }
     })
+    this.$bus.$on('checkout.setShipping', (receivedData) => {
```

Comment:
I created a new event listener, since I needed a way to push the shipping data to the component from the droppoint map component. I couldn't find any events that let me do this.